koinex_db.js

```diff
@@ -11,7 +11,93 @@ var koinex_data = mongoose.Schema({
     collection: 'koinex_data',
     strict: true
 });
+
+var calculated_data = mongoose.Schema({
+    calculated: { type: Array },
+}, {
+    collection: 'calculated_data',
+    strict: false
+});
+
+var BTC = mongoose.Schema({
+    date: { type: String },
```

Comment:
again we are having string type here?   i through previously we have problem with data type itself

Reply:
sir previously it was not working because date was stored inside an array, i was not able to apply the query on it. now it is working absolutely fine